### DISCUSSION TITLE
Can sourcemaps be handled by namer plugins?

### DISCUSSION THREAD

**Discussion by mortendevold** [parcel-bundler/parcel#9094] [General] [answered]
I'm using a custom namer plugin, and it does not seem to handle sourcemap files (they get output to the same location as the CSS and JS files, according to their namer rules). I tried writing a local plugin, but it doesn't look like sourcemap files are passed to it. Can someone please clarify? I ended up writing a small post-build script to handle it, but would prefer not to.

**Comment by mischnic** [comment, marked as answer]
No, they're not handled by namers. They always get written to `correspondingBundleName + ".map"`.
